third_party/bazel_json/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright 2019 Erick Johnson <ejohnson82@gmail.com>
```

Comment:
I've no objections to using this library, nor to committing it in the repo, but:

- `vendor` might be a better name for the top-level directory
- For better Git history, consider doing a subtree merge of the original repo rather than a copy-paste

Neither point is blocker for the PR.